src/nuklear.h

```diff
@@ -203,6 +213,11 @@ NK_STATIC_ASSERT(sizeof(nk_flags) >= 4);
 NK_STATIC_ASSERT(sizeof(nk_rune) >= 4);
 NK_STATIC_ASSERT(sizeof(nk_size) >= sizeof(void*));
 NK_STATIC_ASSERT(sizeof(nk_ptr) >= sizeof(void*));
+#ifdef NK_INCLUDE_STANDARD_BOOL
+NK_STATIC_ASSERT(sizeof(nk_bool) == sizeof(bool));
+#else
+NK_STATIC_ASSERT(sizeof(nk_bool) == 4);
```

Comment:
`nk_bool` is defined as `int` here and `int` is defined in the C standard as having a minimum size of 16 bits, so this assert is incorrect. Hmm, either this assert should be `sizeof(nk_bool) >= 2` or `nk_bool` should be defined as `nk_int` (which would technically be a breaking change)